### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by gracesessions**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -3,24 +3,53 @@
...
-  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+#ham-menu-items li {
+  list-style-type: none;
+}
+
+#ham-nav label,
+ #hamburger {
```
Comment (Gantthebant) [inline css/style.css:37]: On two different lines. Can have it on one

File: css/style.css [hunk 2]
```diff
@@ -135,28 +185,117 @@ a:hover {
...
-}
-
 footer {
   margin: 1rem;
   font-size: 0.75rem;
 }
 
+@media only screen and (max-width:932px) {
+  
+  .content{
```
Comment (Gantthebant) [inline css/style.css]: space needed before {

File: index.html [hunk 3]
```diff
@@ -91,24 +90,25 @@ <h3>Project title</h3>
...
   <footer>
-	  <p>© Copyright 2022</p>
-	  <p><a href="https://www.vecteezy.com/free-vector/programmer">Programmer Vectors by Vecteezy</a></p>
+    <p>© Copyright 2022</p>
+    <p><a href="https://www.vecteezy.com/free-vector/programmer">Programmer Vectors by Vecteezy</a></p>
   </footer>
-
+</div>
+    <a href="#top" id="button">Top <i class="fa-solid fa-chevron-up"></i></a>
 </body>
```
Comment (Gantthebant) [inline index.html:116]: Body needs to be more towards left or move the everything else to the right

**Review by Gantthebant — APPROVED** [review]
In your html line 30 div is spaced too far in. For some reason can't comment it.

File: css/style.css [hunk 4]
```diff
@@ -3,24 +3,53 @@
   font-family: courier, sans-serif;
```
Comment (danieljporter21) [inline css/style.css:3]: is there a space in front of lines 1  and 3?

Comment (danieljporter21) [inline css/style.css:3]: extra space in front of font family?

File: css/style.css [hunk 5]
```diff
@@ -59,6 +98,13 @@ header {
...
+  border-style: solid;
+  border-color: #2E8B57;
+  justify-content: center;
+}
+
+.about:hover {
+  border-color: #FFFFFF;
 }
 
 .about h2 {
```
Comment (danieljporter21) [inline css/style.css:110]: to much space line 111 margin

File: index.html [hunk 6]
```diff
@@ -7,108 +7,110 @@
...
+      <div class="text">
+        <h2>About</h2>
+        <p>Hi, I'm Grace. I recently completed a Bachelors and a Masters degree in Economics.</p>
+        <p>Currently, I am training to be a full-stack software developer at iO-Academy.</p>
+      </div>
+      <div class="grad">
+        <img src="images/me2.jpeg" alt="Grace">
+      </div>
+    </section>
+    <div class="arrow"><a href="#portfolio"><i class="fa-solid fa-chevron-down fa-bounce"></i></a>
```
Comment (danieljporter21) [inline index.html]: maybe put the anchor nested and indented inside the div?

**Review by danieljporter21 — CHANGES_REQUESTED** [review]

**Pull request merged**